dev/AppNotifications/AppNotificationProgressData.h

```diff
@@ -19,7 +19,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         void Status(hstring const& progressStatus);
 
     private:
-        uint32_t m_sequenceNumber{};
+        uint32_t m_sequenceNumber = 1;
```

Comment:
I thought brace initializers were prefered

Reply:
Its a primitive type. So = should be fine. It's mainly preference at the end of the day.